api/src/index.ts

```diff
@@ -16,9 +16,10 @@ import {
   errorHandlingMiddleware,
 } from "@middlewares";
 import "./utils/passaport-strategy";
-
+console.log('Routes import with success')
 const port = process.env.PORT || 7200;
 const app = express();
+console.log('Express App Initialized with success')
```

Comment:
thats console.log is just to see the progress of app in docker terminal, because its take a long time